README.md

```diff
@@ -1,45 +1,102 @@
+HIP stands for "Hedera Improvement Proposal". These improvement proposals can range from core protocol changes, to the applications, frameworks, and protocols built on top of the Hedera public network and used by the community. The HIP author is responsible for building consensus within the community and documenting dissenting opinions, as well as tracking their HIP through the process outlined below. 
 
-HIP is a convenient way for Hashgraph implementers to keep track of their process. While implementing, they will be able to list the HIPs that they are working on or which one they have already implemented. Therefore, this will be an easy way for end users to know the current status of any proposals.
+## What is Hedera Hashgraph? 
 
-## HIP Types
-There are three types of HIP:
-- **A Standard HIP** - describes any changes that affect most or all Hashgraph implementations, this can be a change to the network protocol, a transaction validity rules, proposed application standards/conventions, or any changes or addition that affects the interoperability of applications using Hashgraph. Standards HIPs consist of three parts, a design document, implementation, and an update to the formal specification (if warranted). Furthermore, Standard HIPs can be broken down into the following categories:
-   - **Core:** This includes improvements requiring a consensus fork or as well as changes that are not necessarily consensus critical but may be relevant to core development discussions.
-   - **Networking:** This includes improvements around proposed improvements to network protocol specifications.
-   - **Interface:** This includes improvements around client API/RPC specifications and standards, and also certain language-level standards.
-   - **HRC:**  Hedera Request For Comment. This includes application-level standards and conventions, including contract standards such as token standards, name registries, etc.
+[Hedera Hashgraph](https://hedera.com) is the only public network built on top of [Dr. Leemon Baird](http://www.leemon.com/)’s [Hashgraph consensus algorithm](http://www.leemon.com/papers/2016b.pdf). Hedera goes beyond blockchain to provide the fast, fair, and secure environment needed to enable enterprise adoption of distributed ledger technologies. You can learn more about Hedera by [reading our whitepaper](https://hedera.com/whitepaper), and for a more detailed understanding of the Hashgraph Consensus Algorithm you can check out [it’s whitepaper](http://www.leemon.com/papers/2016b.pdf). 
 
-- **An Informational HIP** - describes an Hashgraph design issue, or provides general guidelines or information to the Hashgraph community, but does not propose a new feature. Informational HIPs do not necessarily represent Hashgraph community consensus or recommendation, so users and implementers are free to ignore Informational HIPs.
+## The purpose of HIPs 
```

Comment:
Think just `Purpose` for a title is sufficient since we're in the context of the HIP repo. Same for most of the below titles.